pages/linux/rename.ul.md

```diff
@@ -0,0 +1,29 @@
+# rename.ul
+
+> Renames multiple files.
+> This is the util-linux version of `rename`. See `rename` for the perl version which is default on Debian.
+> Warning: This command has no safeguards and will overwrite files without prompting.
+
+- Rename files using simple substitutions (substitute 'foo' with 'bar' wherever found):
+
+`rename {{foo}} {{bar}} {{*}}`
+
+- Dry-run - display which renames would occur without performing them:
+
+`rename -vn {{foo}} {{bar}} {{*}}`
+
+- Do not overwrite existing files:
+
+`rename -o {{foo}} {{bar}} {{*}}`
+
+- Change file extensions:
+
+`rename {{.ext}} {{.bak}} *.foo`
```

Comment:
```suggestion
`rename {{.ext}} {{.bak}} {{*.ext}}`
```
This command doesn't make much sense, it changes `.ext` to `.bak`, but the files selected have a `.foo` extension which basically nullifies the purpose of the command.

Reply:
Woops thanks :)